packages/thirdweb/src/utils/encoding/helpers/role-bytes.ts

```diff
@@ -0,0 +1,14 @@
+import { keccak256 } from "../../hashing/keccak256.js";
+import { toBytes } from "../to-bytes.js";
+
+/**
+ * Generates a 256-bit hash of a given role string in bytes form using the keccak256 algorithm.
+ *
+ * @param {string} role - The role string to be converted into bytes and hashed.
```

Comment:
how about a union type for this of 'known' role strings + | ({} & string) to allow for others:
```
type KnownRoles = "ADMIN_ROLE" | "MINTER_ROLE" ...
type Role = KnownRoles | ({} & string)
```